README.md

```diff
@@ -1 +1,7 @@
-# Norman Utils External Package
\ No newline at end of file
+# Norman Utils Overview
+
+Norman Utils is a shared library of helper utilities designed for SDK developers working within the Norman ecosystem.
```

Comment:
Norman Utils is a shared utility library hosting a selection of basic classes and tools used in the Norman ecosystem.

+ Flesh out a bit. This repo is not targeted at developers. Direct them to use the core and high-level SDKs.